src/commands/bonuses-bv/redeem-bonuses.ts

```diff
@@ -0,0 +1,202 @@
+import { Command, flags } from "@oclif/command";
+import { BlobServiceClient } from "@azure/storage-blob";
+import { pickAzureConfig, getStorageConnection } from "../../utils/azure";
+import fetch from "node-fetch";
+import cli from "cli-ux";
+import { DateTime } from "luxon";
+
+interface OutputResult {
+  redeemedRequest: string;
+  statusCode: string;
+}
+
+export default class RedeemBonuses extends Command {
+  public static description = "Reprocess redeemed bonuses in the specified day";
+
+  public static args = [];
+
+  public static flags = {
+    day: flags.string({
+      description:
+        "filter reddem request from specified day (utc timezome, required format yyyy-MM-dd, ie 2020-05-25)",
+      required: true
+    }),
+    fromTime: flags.string({
+      description:
+        "filter reddem request from specified time (utc timezone, required format HH:mm, ie 09:45)"
+    }),
+    toTime: flags.string({
+      description:
+        "filter reddem request to specified time (utc timezone, required format HH:mm, ie 15:35)"
+    }),
+    apiUrl: flags.string({
+      default: "https://api.io.italia.it/api/bonus-vacanze/v1/redeemed",
+      description: "bonus vacanze api url"
+    }),
+    apiKey: flags.string({
+      description: "bonus vacanze api key",
+      required: true
+    })
+  };
+
+  public async run(): Promise<void> {
+    const { flags: parsedFlags } = this.parse(RedeemBonuses);
+
+    const day = DateTime.fromFormat(
+      parsedFlags.day + " " + "utc",
+      "yyyy-MM-dd z"
+    );
+    if (!day.isValid) {
+      this.error("day is not valid");
+      return;
+    }
+
+    let fromTime = undefined;
+    if (parsedFlags.fromTime !== undefined) {
+      fromTime = DateTime.fromFormat(
+        parsedFlags.day + " " + parsedFlags.fromTime + " " + "utc",
```

Comment:
```suggestion
        `${parsedFlags.day} ${parsedFlags.fromTime} utc`,
```
[it's the same](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) but it's more readable

Reply:
fixed, thank you!